server/src/controllers/seasonController.ts

```diff
@@ -145,6 +145,96 @@ seasonController.get('/', async (req: Request, res: Response) => {
   }
 });
 
+/**
+ * Retrieves a list of seasons with their associated events.
+ *
+ * @swagger
+ * /2/season/list:
+ *   get:
+ *     summary: Get all seasons with events
+ *     tags:
+ *     - Season
```

Comment:
The tag should be `New season` so that it is in the same place as the rest of the season APIs
![image](https://github.com/WonderTix/WonderTix/assets/92124260/a94606e8-bfd0-42eb-9d66-305a8dc3d4c1)

Reply:
fixed